org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -88,6 +88,152 @@ class Translator {
         // Make the async request
         return this._makeRequest(requestPath, 'PUT', putPayloadString);
     }
+    
+    /**
+     * Subscribes to a Wink pubsubhubbub feed.  This function is designed to be called twice.
+     * The first call will contain just the callbackUrl, which will be subscribed for postbacks
+     * from the Wink cloud service.
+     * The second call will contain the request, and response for a GET to the callbackUrl that was
```

Comment:
It's what we will be going with (at least in the short term) in order to support cloud-to-cloud notifications.  PubsubHubbub requires a handshake process from the callback server, but the goal of the translators is to hide the protocol we use from anybody using the translators, so they cannot perform that process themselves.  In the future, we'll have to make significant modifications to this in order to support services that offer non cloud-to-cloud notifications, but for now, this allows customers to begin using Wink devices.

---
In reply to: [80521349](https://github.com/openT2T/translators/pull/91#discussion_r80521349) [](ancestors = 80521349)